src/noun.substance

```diff
@@ -469,6 +469,22 @@ hs: limestone
 hyper: calcium_ion
 hypo: metal
 
+w: Calciarenite
+d: [lithology] A limestone consisting predominantly (> 50%) of rounded detrital calcite particles (either bigenetic or non-biogenetic) between 0.063 and 2 mm. Interstices are filled with carbonate mud or cement (Füchtbauer, 1988; Neuendorf et al., 2005).
```

Comment:
vamos tirar este [lithology] do inicio das descrições.

Reply:
Ok, de acordo. Coloquei assim por ter identificado alguns [geology], achei que poderia ajudar futuramente.

Ok, agree to remove [lithology]. I used it inspired by [geology], already used in some cases,

Reply:
os casos que vc achar de [geology] seria bom verificar a existência da relação domain com geology e então remover este [geology] e note que estavam com parênteses não com colchetes.

Reply:
Já fiz isso para lithology, posteriormente faço para geology